egs/multi_en/s5/local/nnet3/run_tdnn.sh

```diff
@@ -0,0 +1,109 @@
+#!/bin/bash
+
+###########################################################################################
+# This script was copied from egs/fisher_swbd/s5/local/nnet3/run_tdnn.sh
+# The source commit was e69198c3dc5633f98eb88e1cdf20b2521a598f21
+# Changes made:
+#  - Modified paths to match multi_en naming conventions
+###########################################################################################
+
+# this is the standard "tdnn" system, built in nnet3; it's what we use to
+# call multi-splice.
+
+. ./cmd.sh
+
+
+# At this script level we don't support not running on GPU, as it would be painfully slow.
+# If you want to run without GPU you'd have to call train_tdnn.sh with --gpu false,
+# --num-threads 16 and --minibatch-size 128.
+
+stage=0
+affix=
+train_stage=-10
+common_egs_dir=
+reporting_email=
+remove_egs=true
+multi=multi_a
+
+. ./cmd.sh
+. ./path.sh
+. ./utils/parse_options.sh
+
+
+if ! cuda-compiled; then
+  cat <<EOF && exit 1
+This script is intended to be used with GPUs but you have not compiled Kaldi with CUDA
+If you want to use GPUs (and have them), go to src/, and configure and make on a machine
+where "nvcc" is installed.
+EOF
+fi
+
+dir=exp/$multi/nnet3/tdnn
+dir=$dir${affix:+_$affix}
+train_set=$multi/tdnn
+ali_dir=exp/$multi/tri5_ali_sp
+
+local/nnet3/run_ivector_common.sh --stage $stage --multi $multi \
+  --speed-perturb false || exit 1;
+
+exit 0; # STOP
+
+if [ $stage -le 9 ]; then
+  echo "$0: creating neural net configs";
+
+  # create the config files for nnet initialization
+  python steps/nnet3/tdnn/make_configs.py  \
+    --feat-dir data/${train_set}_hires \
+    --ivector-dir exp/$multi/nnet3/ivectors_${train_set} \
+    --ali-dir $ali_dir \
+    --relu-dim 1024 \
+    --splice-indexes "-2,-1,0,1,2 -1,2 -3,3 -3,3 -7,2 0"  \
+    --use-presoftmax-prior-scale true \
+   $dir/configs || exit 1;
+fi
+
+
+if [ $stage -le 10 ]; then
+  if [[ $(hostname -f) == *.clsp.jhu.edu ]] && [ ! -d $dir/egs/storage ]; then
+    utils/create_split_dir.pl \
+     /export/b0{3,4,5,6}/$USER/kaldi-data/egs/fisher_swbd-$(date +'%m_%d_%H_%M')/s5/$dir/egs/storage $dir/egs/storage
```

Comment:
Yes.


在 2016/7/28 3:51, Allen Guo 写道:
>
> In egs/multi_en/s5/local/nnet3/run_tdnn.sh 
> <https://github.com/kaldi-asr/kaldi/pull/771#discussion_r72510268>:
>
> > +  # create the config files for nnet initialization
> > +  python steps/nnet3/tdnn/make_configs.py  \
> > +    --feat-dir data/${train_set}_hires \
> > +    --ivector-dir exp/$multi/nnet3/ivectors_${train_set} \
> > +    --ali-dir $ali_dir \
> > +    --relu-dim 1024 \
> > +    --splice-indexes "-2,-1,0,1,2 -1,2 -3,3 -3,3 -7,2 0"  \
> > +    --use-presoftmax-prior-scale true \
> > +   $dir/configs || exit 1;
> > +fi
> > +
> > +
> > +if [ $stage -le 10 ]; then
> > +  if [[ $(hostname -f) == *.clsp.jhu.edu ]] && [ ! -d $dir/egs/storage ]; then
> > +    utils/create_split_dir.pl \
> > +     /export/b0{3,4,5,6}/$USER/kaldi-data/egs/fisher_swbd-$(date +'%m_%d_%H_%M')/s5/$dir/egs/storage $dir/egs/storage
>
> You mean instead of |fisher_swbd| right?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub 
> <https://github.com/kaldi-asr/kaldi/pull/771/files/730c4fd2377d13503ae472ff72d456313dd85a5b..65858d4060caff6e8761a382c4584916023cecb3#r72510268>, 
> or mute the thread 
> <https://github.com/notifications/unsubscribe-auth/ADKpxPA0q5ui78jX3_cIaZaItsJzyxsKks5qZ7angaJpZM4IZ3x7>.
>